shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -1333,7 +1333,16 @@ export default {
 
         // if there's the initial machine pool config, we are in a good position to apply the handleConflict function
         // to deal with out-of-sync data between machinePools configs. This also mutates the data inside machinePool.config through object reference
-        const conflict = await handleConflict(initialMachinePoolValue.toJSON(), machinePool.config, latestConfig, this.$store.getters, this.$store, 'management');
+        const conflict = await handleConflict(
+          initialMachinePoolValue,
+          machinePool.config,
+          latestConfig,
+          {
```

Comment:
You are changing the functions signature here (one less argument, now the toJSON is inside the actual function) which will cause problems with backwards compatibility IF there's an extension that uses `handleConflict`... I would say the chances of that are low, but it's good to know the risks involved. FYI @richard-cox

Reply:
Let's double check if that code is used in extensions

Reply:
Harvester, Elemental, Kubewarden don't have reference to `handleConflicts`,